tests/functional/spec/features/repeating_blocks/list_collector_repeating_blocks.spec.js

```diff
@@ -87,57 +89,48 @@ describe("List Collector Repeating Blocks", () => {
 
-    it("The list collector shows all of the companies.", async () => {
-      const companiesExpected = ["ONS", "GOV", "MOD"];
-      checkItemsInList(companiesExpected, AnyOtherCompaniesOrBranchesPage.listLabel);
-    });
-
-    it("The list collector allows the name of 'GOV' to be changed", async () => {
+    it("When the user edits an item, Then the name of the item is able to be changed", async () => {
       await $(AnyOtherCompaniesOrBranchesPage.listEditLink(2)).click();
       await $(EditCompanyPage.companyOrBranchName()).setValue("Government");
       await click(EditCompanyPage.submit());
       await expect(await $(AnyOtherCompaniesOrBranchesPage.listLabel(2)).getText()).toBe("Government");
     });
 
-    it("The list collector allows removal of 'Government'", async () => {
+    it("When the user clicks the remove link, Then the item selected is removed", async () => {
       await $(AnyOtherCompaniesOrBranchesPage.listRemoveLink(2)).click();
       await $(RemoveCompanyPage.yes()).click();
       await click(RemoveCompanyPage.submit());
-    });
-
-    it("The list collector does not show 'GOV' anymore.", async () => {
+      checkItemsInList(["ONS", "MOD"], AnyOtherCompaniesOrBranchesPage.listLabel);
       await expect(await $(AnyOtherCompaniesOrBranchesPage.listLabel(2)).getText()).not.toContain("Government");
       await expect(await $(AnyOtherCompaniesOrBranchesPage.listLabel(2)).getText()).toBe("MOD");
     });
 
-    it("The list collector can add more companies.", async () => {
+    it("When a user has finished editing or removing from the list, Then they are still able to add additional companies", async () => {
       await $(AnyOtherCompaniesOrBranchesPage.yes()).click();
       await click(AnyOtherCompaniesOrBranchesPage.submit());
       await addCompany("Council", "101", "4", "4", "2023", false, true);
+      checkItemsInList(["ONS", "MOD", "Council"], AnyOtherCompaniesOrBranchesPage.listLabel);
     });
 
-    it("The list collector shows all of the companies.", async () => {
-      const companiesExpected = ["ONS", "MOD", "Council", "another one"];
-      checkItemsInList(companiesExpected, AnyOtherCompaniesOrBranchesPage.listLabel);
-    });
-
-    it("The list collector can then be submitted", async () => {
+    it("When a user has finished making changes to the list, Then section can be completed and the questionnaire submitted", async () => {
       await $(AnyOtherCompaniesOrBranchesPage.no()).click();
       await click(AnyOtherCompaniesOrBranchesPage.submit());
 
       await click(AnyOtherTradingDetailsPage.submit());
       await click(SectionCompaniesPage.submit());
       await click(SubmitPage.submit());
+      await expect(await browser.getUrl()).toContain(ThankYouPage.pageName);
     });
   });
 
   describe("Given a journey through the list collector with repeating blocks, any incomplete repeating_block will be revisited before navigating to the next block after the list collector.", () => {
```

Comment:
`Given a journey that tests routing through....` ? Totally open to other thoughts, but I reckon these could be a bit more consistent, the other two are very generic and this one is quite specific.

Reply:
Amended slightly now 👍